### ISSUE TITLE
Database setup and installation help

### ISSUE THREAD

**Issue by hasangursoy** [waqaskhan540/QuestionAnswerApp#9]
Would be nice if readme contains configuration information.

**Comment by waqaskhan540** [comment]
Yes you are right. I have just started building it, I will add the documentation and configuration details asap. Thanks for the suggestion.

**Comment by waqaskhan540** [comment]
@hasangursoy I have updated the readme, let me know if you need more clarification.
